tests/executable/test space.tex

```diff
@@ -0,0 +1,4 @@
+\documentclass{article}
```

Comment:
i would prefer if this test stuck to plain TeX, rather than LaTeX, so as not to require a full download of all of the LaTeX support files to run it. In principle it could even pass a `--format` argument to `tectonic` so that it could use the plain format file generated in the `tex-outputs` test, which can be generated without touching the network at all. But I am OK with a version that needs to touch the network — which will still need a `--format` flag to change the standalone executable away from its default of using the `latex` format.